src/sentry/testutils/fixtures.py

```diff
@@ -668,6 +668,12 @@ def create_data_condition_group_action(self, *args, **kwargs):
     def create_detector_workflow(self, *args, **kwargs):
         return Factories.create_detector_workflow(*args, **kwargs)
 
+    def create_alert_rule_detector(self, *args, **kwargs):
```

Comment:
mind adding some TODO's here in code so we can remind future us that this is only needed during the migration?